src/components/Practice2.tsx

```diff
@@ -0,0 +1,16 @@
+import { useState } from 'react';
+import { Input } from './input';
+
+export const Practice2 = () => {
+  const [isChecked, setIsChecked] = useState(false);
```

Comment:
From [React Guideline](https://pj100.esa.io/posts/16722#%E3%82%A2%E3%83%B3%E3%83%81%E3%83%91%E3%82%BF%E3%83%BC%E3%83%B3%E3%81%AE%E6%94%B9%E5%96%84), useState should have predefined type like so:

```suggestion
  const [isChecked, setIsChecked] = useState<boolean>(false);
```